### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by NotNestor**
## Description
* Remove node-sass dependency which has been deprecated and is preventing update to later version of node

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,8 +13,7 @@ jobs:
     steps:
       - checkout
       - run: yarn global add @percy/agent lerna@4.0.0
-      - run: yarn bootstrap
-      - run: npm rebuild node-sass
+      - run: yarn bootstrap
```
Comment (NotNestor) [inline .circleci/config.yml:16]: sass does not need to rebuilt like node-sass as it is a javascript implementation

File: core/package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "ci:publishCDN": "run-s replaceFontURLs pushToS3",
     "pushToS3": "node ../tools/pushToS3.js --library=core",
     "replaceFontURLs": "node ./tools/replaceURLs.js",
-    "css:build": "node-sass -i templates/momentum-ui.scss -o css/ --importer=../node_modules/node-sass-tilde-importer",
+    "css:build": "node ./tools/compileCSSMomentum.js",
     "css:build:components": "node ./tools/compileCSSModules.js",
```
Comment (NotNestor) [inline core/package.json:45]: sass does not have an --importer param for cli so create js script

File: .circleci/config.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ definitions:
   image_defaults: &image_defaults
     working_directory: ~/repo
     docker:
-      - image: cypress/included:8.7.0
+      - image: cypress/included:13.13.1
         environment:
```
Comment (NotNestor) [inline .circleci/config.yml:6]: move from node 14 to node 20 build

File: web-components/package.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
     "start": "npm run dev",
     "dev": "webpack-dev-server --config ./webpack.config.dev.server.ts",
     "dev:build": "webpack --config-name=dev",
-    "dist": "webpack --config-name=distProd",
+    "dist": "NODE_OPTIONS=--openssl-legacy-provider webpack --config-name=distProd",
     "dist:dev": "webpack --config-name=distDev",
```
[the remark this answers is not in the document]
Comment (ctummon) [reply web-components/package.json:14]: This wont build on Windows, we've been using `cross-env` to get around it in the wxcc-desktop repo

File: core/tools/compileCSSModules.js [hunk 5]
```diff
@@ -59,7 +59,7 @@ const modules = [
 ];
 
 const compileCSSModules = () => {
-  modules.forEach((module, idx) => {
+  modules.forEach(module => {
```
Comment (ctummon) [inline core/tools/compileCSSModules.js:62]: Maybe this file is stale but the linter seems to be making modifications unrelated to your changes, as long as its expected that's fine!

File: web-components/package.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (NotNestor) [reply web-components/package.json:14]: 👍

**Pull request merged**